scripts/exglobal_atmos_analysis.sh

```diff
@@ -359,14 +367,26 @@ cd ${DATA} || exit 99
 # Fixed files
 ${NLN} ${BERROR}       berror_stats
 ${NLN} ${SATANGL}      satbias_angle
-${NLN} ${SATINFO}      satinfo
+if [[ ${SATINFO} == "generate" ]]; then
+   sh ${USHgfs}/create_satinfo.sh  ${PDY}${cyc} > satinfo
```

Comment:
Prefer that the `create_satinfo.sh` file be called directly.  If it isn't already, that script should be made executable.

```suggestion
   ${USHgfs}/create_satinfo.sh  ${PDY}${cyc} > satinfo
```

Reply:
done